dockerfiles/jupyter/align_mlrun.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+for env in base jupyter; do
```

Comment:
I am not sure it is relevant. on mlrun's jupyter we ship `mlrun` and `base` env.
the reason. in addition, I dont think the persistent comment here is relevant at all. need to double verify which env is persisnt and which not on our default mlrun's ce deployment and take an action accordingly